npm/css-src/sass/_card.scss

```diff
@@ -1,3 +1,5 @@
+//This card component can be replaced by the '_card-metadata.scss' when the 'card' component on the additional metadata start page is updated
```

Comment:
Could you check with Gareth whether the "card components" scss will be the same across all the pages. If so might aswell make this change in this PR.

Reply:
Confirmed with Gareth that the card components scss will be the same across all the pages.